frontend/app/forms/proof-of-completed-service/proof-of-completed-service.web.component.js

```diff
@@ -161,7 +161,107 @@ export default function ProofOfCompletedService_Web({ data }) {
+      <Section name="4. Service by Third Person (continued)">
+        <Checkbox
+          dataKey="case.proofOfCompletedServiceDeliveredForm"
+          label="On"
+        />
+        <TextInput
+          dataKey="person.proofOfCompletedServiceDelieveredDate"
+          label="date MM-DD-YYYY"
+        />
+        <p>I went to</p>
+        <TextInput
+          dataKey="person.proofOfCompletedServiceDelieveredAddressZip"
+          label="(address)"
+        />
+        <p>and I delivered the documents listed in paragraph 1 to</p>
+        <TextInput
+          dataKey="person.proofOfCompletedServiceDelieveredName"
+          label="(name)"
+        />
+        <p>
+          who is <small>(Check one.)</small>
+        </p>
+        <Checkbox
+          dataKey="person.defendant"
+          label="the named defendant/respondent"
+        />
+        <Checkbox
+          dataKey="person.plaintiff"
+          label="the named plaintiff/petitioner"
+        />
+        <Checkbox
+          dataKey="person.proofOfCompletedServicePersonSuitableAge"
+          label="a person of suitable age and discretion residing at that address, which is the named party's residence"
+        />
+        {data.person.proofOfCompletedServicePersonSuitableAge && (
+          <TextArea
+            dataKey="person.proofOfCompletedServicePersonSuitableAgeDescribe"
+            label="Describe why the person lives at the named party's residence and why they are of suitable age and discretion"
+          />
+        )}
+
+        <Checkbox
+          dataKey="person.agent"
+          label="an agent authorized by appointment or by law to receive servie of process on behalf of the named party"
+        />
+        <br></br>
+        <small>
+          If serving a corportion, partnership, or an unincorporated association
+        </small>
+        <Checkbox
+          dataKey="person.officer"
+          label="an officer, a managing agent, general agent"
+        />
+        <br></br>
+        <strong>OR</strong>
+        <Checkbox
+          dataKey="person.authorizedAgent"
+          label="an agent authorized by appointment or by law to receive service of
+          process and by also mailing a copy of the complaint and summons to the
+          named party, if the agent is one authorized by statute to receive process
+          and the statute so requires."
+        />
+        <br></br>
+        <small>If serving a city or town</small>
+        <Checkbox dataKey="person.recorder" label="the city/town recorder" />
+        <br></br>
+        <small>If serving a county</small>
+        <Checkbox dataKey="person.clerk" label="the county clerk" />
+        <br></br>
+        <small>If serving the state</small>
+        <Checkbox
+          dataKey="person.attorneyGeneral"
+          label="the attorney general, and to"
+        />
+        <TextInput
+          dataKey="person.proofOfCompletedServicePersonStatue"
```

Comment:
```suggestion
          dataKey="person.proofOfCompletedServicePersonStatute"
```

Please fix the other typos where it says Statue instead of Statute

Reply:
Good catch!

Reply:
Corrected any/all instance of statute.